test/run-deps

```diff
@@ -65,6 +65,16 @@ testPylibmc() {
   assertCapturedSuccess
 }
 
+testDeployingTwiceUsesTheCache() {
+  compile "pythonDefault"
+  assertNotCaptured "Installing collected packages:"
+  assertCapturedSuccess
+  compile "pythonDefault"
+  assertCaptured "Requirement already satisfied:"
+  assertNotCaptured "Installing collected packages:"
+  assertCapturedSuccess
+}
+
```

Comment:
Did this end up being useful? I remember it passing even without our patch? We've got the hatchet test as well.